NEWS.md

```diff
@@ -1,3 +1,8 @@
+### Accessibility
+
+* Added appropriate labels to `icon()` element to provide screen-reader users with alternative descriptions of the `fontawesome`: `aria-label` is automatically applied based on the fontawesome name. For example, `icon("calendar")` will be announced as "calendar icon" to screen readers now.
```

Comment:
The label also applies to glyphicon, right?

Reply:
Yes, correct! I have just pushed the updated `NEWS.md`.